oaknational/Oak-Web-Application
# feat: keystage keypad component

## Description

- Adds key stage keypad component
- Adds to storybook
- Adds tests
- Adds subject by key stage slug page. /beta/key-stages/key-stage-1 etc

Design: https://www.figma.com/file/hQoMOEvJQlDJZ0n2ehftVT/Beta?node-id=924%3A381743

## Issue(s)

Progresses #1016

## How to test

1. Go to storybook deployment navigation/jkeystage keypad
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
